man/journald.conf.xml

```diff
@@ -139,6 +139,50 @@
         <literal>us</literal>. To turn off any kind of rate limiting,
         set either value to 0.</para>
 
+        <para>The <varname>RateLimitBurst=</varname> is further modified
+        by the log() of the available disk space for the journal.</para>
```

Comment:
Maybe just say "logarithm"? "log" is easy to confuse with "logging" in this context.

Reply:
should say to which base

Reply:
I used () to imply a function but I can spell it out.